src/components/Button/Button.module.scss

```
@@ -0,0 +1,27 @@
+@import "styles/variables";
+
+.buttonMain {
+  padding: $space-s $space-l;
+  display: flex;
+  justify-content: center;
+  align-items: center;
```

советую подобные штуки (наборы правил, которые много где используются) выносить в миксины. в данном случае можно, например, создать миксин
```scss
@mixin flex-center {
  display: flex;
  justify-content: center;
  align-items: center;
}
```
и использовать тут
```suggestion
  @include flex-center;
```
и в остальных местах где надо отцентровать контент